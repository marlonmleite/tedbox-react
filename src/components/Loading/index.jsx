import React from 'react'
import CircularProgress from '@material-ui/core/CircularProgress';
import LoadingStyled from './styled'

const Loading = () => (
  <LoadingStyled>
    <CircularProgress />
  </LoadingStyled>
)

export default Loading
