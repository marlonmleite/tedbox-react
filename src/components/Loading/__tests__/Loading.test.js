import React from 'react'
import { shallow } from 'enzyme'
import Loading from '..'

describe('Loading', () => {
  it('should render', () => {
    const component = shallow(<Loading />)

    expect(component).toMatchSnapshot()
  })
})

