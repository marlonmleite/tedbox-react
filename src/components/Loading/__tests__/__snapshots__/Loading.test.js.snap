// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render 1`] = `
<styled.div>
  Loading...
</styled.div>
`;
